components/pcloud/pcloud.app.mjs

```diff
@@ -1,11 +1,327 @@
+import get from "lodash/get.js";
+import retry from "async-retry";
+import pcloudSdk from "pcloud-sdk-js";
+
 export default {
   type: "app",
   app: "pcloud",
-  propDefinitions: {},
+  propDefinitions: {
+    fileId: {
+      type: "integer",
+      label: "File ID",
+      async options() {
+        return await this.getFileOptions();
```

Comment:
`return await` is [not necessary](https://eslint.org/docs/latest/rules/no-return-await) unless it's inside a `try/catch` statement; you can just return the value directly.

Reply:
Fixed this for all the `async options`, indeed not needed at all, thanks!